Makefile

```diff
@@ -111,42 +123,48 @@ push-tagged-image: push-image
 # Generate manifests e.g. CRD, RBAC etc.
 manifests: manifests-k8s manifests-ocp
 
-manifests-k8s: generate-crd kustomize
+manifests-k8s: manifests-k8s-crd manifests-k8s-csidriver
+	cp "$(KUBERNETES_CRD_YAML)" "$(KUBERNETES_OLM_YAML)"
+	cat "$(KUBERNETES_CRD_YAML)" "$(KUBERNETES_CSIDRIVER_YAML)" > "$(KUBERNETES_ALL_YAML)"
+
+manifests-k8s-crd: generate-crd controller-gen kustomize
 	# Create directories for manifests if they do not exist
 	mkdir -p config/deploy/kubernetes
 
-	# Generate kubernetes.yaml
+	# Generate kubernetes-crd.yaml
 	helm template dynatrace-operator config/helm/chart/default \
 		--namespace dynatrace \
 		--set platform="kubernetes" \
 		--set manifests=true \
 		--set olm="${OLM}" \
 		--set autoCreateSecret=false \
-		--set operator.image="$(MASTER_IMAGE)" > config/deploy/kubernetes/kubernetes.yaml
-	grep -v 'app.kubernetes.io/managed-by' config/deploy/kubernetes/kubernetes.yaml > config/deploy/kubernetes/tmp.yaml
-	grep -v 'helm.sh' config/deploy/kubernetes/tmp.yaml > config/deploy/kubernetes/kubernetes.yaml
-	rm config/deploy/kubernetes/tmp.yaml
+		--set operator.image="$(MASTER_IMAGE)" > "$(KUBERNETES_OTHERS_YAML)"
+
+	sed -i "/app.kubernetes.io\/managed-by/d" "$(KUBERNETES_OTHERS_YAML)" 
+	sed -i "/helm.sh/d" "$(KUBERNETES_OTHERS_YAML)" 
```

Comment:
`sed -i` does not work on Mac

Reply:
Marking as resolved to make only single instance of this comment visible.